GyroData/Domain/Entities/MotionInformation.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MotionData.swift
+//  GyroData
+//
+//  Created by 이호영 on 2022/12/29.
+//
+
+import Foundation
+
+struct MotionInformation {
+    let id: UUID
+    let motionType: MotionType
+    let date: Date
+    let time: Double
+    let xData: [Double]
+    let yData: [Double]
+    let zData: [Double]
+}
```

Comment:
```suggestion
struct MotionInformation {
    let motion: Motion
    let xData: [Double]
    let yData: [Double]
    let zData: [Double]
}
```

이러면 사용하기 너무 불편할까요? 🤔

Reply:
오~ 좋은 것 같아요!
매핑할때 불편함이 있을수도 있지만... 불편하면 그때 다시 바꾸면되져! 
수정해보겠습니다 ☺️